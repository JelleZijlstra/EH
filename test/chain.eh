#!/usr/bin/ehi

include '../lib/library.eh'

private it = Iterable.chain('foo', 'bar', 'baz')

for i in it
	echo i
end

echo(it.reverse())
