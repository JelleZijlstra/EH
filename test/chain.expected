%%stdout%%
f
o
o
b
a
r
b
a
z
zabraboof
%%stderr%%
